ddtrace/internal/utils/formats.py

```diff
@@ -160,3 +166,26 @@ def stringify_cache_args(args):
             break
 
     return " ".join(out)
+
+
+def _w3c_format_unknown_propagated_tags(current_tags, potential_tags):
```

Comment:
nit: I am not sure if this module is the best place for this function. Maybe we can should move it to `ddtrace/internal/utils/http.py`?
@Kyle-Verhoog  thoughts?

Reply:
I wasn't sure either, I thought about http.py as well but didn't want it to necessarily be only for http propagation? Since we use it in other instances, such as for your dbm work.

Reply:
I'll put in http.py